src/components/users/add-user-dialog.vue

```diff
@@ -0,0 +1,128 @@
+<template>
+  <v-dialog v-model="dialog" max-width="400">
+    <template #activator="{ on }">
+      <slot name="activator" :on="on" />
+    </template>
+    <v-card outlined>
+      <v-card-title>
+        Add User
+      </v-card-title>
+      <v-card-text>
+        <v-text-field
+          v-model.trim="email"
+          label="Email"
+          outlined
+          :error-messages="emailErrors"
+          @blur="$v.email.$touch()"
+        />
+        <v-text-field
```

Comment:
How is this going to work for emails to John etc?

Don't we need some kind of visitor / staff / student number to make that work even if the person doesn't have an `@...uwa.edu.au` email address

Reply:
the username is the pheme number, or whatever john needs. the email is only really for our system